From 2239e4f5696ef65dfd54be42cc31485eb5e78467 Mon Sep 17 00:00:00 2001
From: Kito Cheng <kito.cheng@sifive.com>
Date: Tue, 1 Dec 2020 15:39:09 +0800
Subject: [PATCH 29/60] RISC-V: Drop some commited accidentally code.

gcc/ChangeLog:

	* config.gcc (riscv*-*-*): Drop some commited accidentally code.
---
 gcc/config.gcc | 1 -
 1 file changed, 1 deletion(-)

diff --git a/gcc/config.gcc b/gcc/config.gcc
index bf621cf76c7..ad27ab4a384 100644
--- a/gcc/config.gcc
+++ b/gcc/config.gcc
@@ -4562,7 +4562,6 @@ case "${target}" in
 			exit 1
 			;;
 		esac
-		with_arch=`${srcdir}/config/riscv/arch-canonicalize ${with_arch}`
 		tm_defines="${tm_defines} TARGET_RISCV_DEFAULT_ARCH=${with_arch}"
 
 		# Make sure --with-abi is valid.  If it was not specified,
-- 
2.25.1

